docs/_content/ontwikkelaars/tutorials/notificeren.md

```diff
@@ -0,0 +1,356 @@
+---
+title: Tutorial notificeren
+---
+
+In deze tutorial configureren we de referentieimplementaties van de ZGW
+componenten voor het versturen en ontvangen van notificaties via de
+notificatierouteringcomponent (NRC).
+
+De tutorial is hands-on - onderaan vind je verdere referenties en bronnen
+indien je meer wil lezen.
+
+De volgende componenten zijn meest relevant:
+
+* NRC: voor het routeren van notificaties, kanaal- en abonnementbeheer
+* ZRC: voor het versturen van `zaken`-notificaties
+* DRC: voor het versturen van `documenten`-notificaties
+* ZTC: voor validaties bij het aanmaken van zaken en documenten
+
+## Wat zijn de vereisten voor deze tutorial?
+
+* `docker` en `docker-compose` om lokaal op je (ontwikkelmachine) de
+  componenten te hosten. Zie ['aan de slag'](../aan-de-slag) voor een
+  uitgebreide beschrijving.
+
+* Het handigste is om de containers in 1 command prompt te hebben draaien, en
+  extra commando's in een tweede prompt ernaast uit te voeren. Zorg dat beide
+  prompts zich in de juiste directory bevinden: `/pad/naar/gemma-zaken/infra`.
+
+* Familiariteit met webhooks is een plus
+
+## Eenmalige setup na het opstarten van de containers
```

Comment:
Hmm, dit kan wel naar een generiek stuk. Je moet dit bijna voor wat je ook gaat doen goed inrichten.

Reply:
Ja, is wel een erg lijvig stuk. Na API-lab verhuizen?

Reply:
Is het niet handig als we dit als startpunt in een apart docje gooien VOOR het API-lab?

Reply:
ah op die manier, ik splits het!